tap_jira/http.py

```diff
@@ -284,6 +292,19 @@ def test_basic_credentials_are_authorized(self):
         # Assign True value to is_on_prem_instance property for on-prem Jira instance
         self.is_on_prem_instance = self.request("users","GET","/rest/api/2/serverInfo").get('deploymentType') == "Server"
 
+    def _write_config(self, token):
```

Comment:
`token` param can be removed if we read from instance variables

Reply:
Yes, fixed this.